### PR TITLE
removed break condition

### PR DESCRIPTION
**Pull request by mejango**
# Description

*What does this PR: do, how, why?*

## Limitations & risks

*Are there any trade-off or new vulnarbility surface based on theses changes?*

# Check-list
- [ ] Tests are covering the new feature
- [ ] Code is [natspec'd](https://docs.soliditylang.org/en/v0.8.17/natspec-format.html)
- [ ] Code is [linted and formatted](https://docs.soliditylang.org/en/v0.8.17/style-guide.html)
- [ ] I have run the test locally (and they pass)
- [ ] I have rebased to the latest main commit (and tests still pass)

# Interactions
These changes will impact the following contracts:
- Directly:

- Indirectly:

### CODE REVIEW

File: src/REVLoans.sol [hunk 1]
```diff
@@ -641,7 +617,11 @@ contract REVLoans is ERC721, ERC2771Context, IREVLoans, Ownable {
         // Accept the funds that'll be used to pay off loans.
         amount = _acceptFundsFor({token: loan.source.token, amount: amount, allowance: allowance});
 
+<<<<<<< Updated upstream
```
[the remark this answers is not in the document]
Comment (mejango) [reply src/REVLoans.sol:620]: woops. fixed

File: src/REVLoans.sol [hunk 2]
```diff
@@ -491,32 +483,27 @@ contract REVLoans is ERC721, ERC2771Context, IREVLoans, Ownable {
...
-                newLastLoanIdLiquidated = loanId;
-                continue;
-            }
+            if (owner == address(0)) { continue; }
 
-            // If the loan has not yet passed its liquidation timeframe, no subsequent loans have either.
-            if (block.timestamp <= loan.createdAt + LOAN_LIQUIDATION_DURATION) {
-                break;
-            }
+<<<<<<< Updated upstream
```
[the remark this answers is not in the document]
Comment (mejango) [reply src/REVLoans.sol:501]: fixed

File: src/REVLoans.sol [hunk 3]
```diff
@@ -491,32 +483,27 @@ contract REVLoans is ERC721, ERC2771Context, IREVLoans, Ownable {
...
 
             // If the loan is already burned, continue.
-            if (owner == address(0)) {
-                newLastLoanIdLiquidated = loanId;
-                continue;
-            }
+            if (owner == address(0)) { continue; }
 
-            // If the loan has not yet passed its liquidation timeframe, no subsequent loans have either.
-            if (block.timestamp <= loan.createdAt + LOAN_LIQUIDATION_DURATION) {
```
[the remark this answers is not in the document]
Comment (mejango) [reply src/REVLoans.sol:516]: whoops again. jeeze this was a bad commit. all is fixed now (i think) :]

**Comment by mejango** [comment]
need slight test touchup @simplemachine92

**Review by simplemachine92 — APPROVED** [review]
lgtm

**Pull request merged**